operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,551 @@
+Specifying these prefixes will create the zones for `Kubernetes` and `Ceph` accordingly with the prefixes added to the zone labels.
+
+#### Procedure
+
+Update the `customizations.yaml` file before commencing the CSM installation or upgrade.
+
+##### `Kubernetes` zone prefix updation
+
+Update the `spec.services.k8s_zone_prefix` section in the `customizations.yaml` file with the required `Kubernetes` zone prefix.
+
+Please follow similar steps [Enabling Rack Resiliency feature](#1-enabling-rack-resiliency-feature) in order to update and apply above customization changes.
+
+##### `Ceph` zone prefix updation
+
+Update the `spec.services.Ceph_zone_prefix` section in the `customizations.yaml` file with the required `Ceph` zone prefix.
+
+Please follow similar steps [Enabling Rack Resiliency feature](#1-enabling-rack-resiliency-feature) in order to update and apply above customization changes.
+
+**Imortant Nodes:**
+* If the prefixes is defined, then the zones will be created in the format of `k8s_zone_prefix + rack_id` and `ceph_zone_prefix + rack_id`.
+    * If the `spec.services.k8s_zone_prefix` has a value of `test-system` and the rack-id is `x3000`, then the `Kubernetes` zones will be created with the labels of value `test-system-x3000`.
+    * If the `spec.services.ceph_zone_prefix` has a value of `test-storage-system` and the rack-id is `x3000`, then the `Ceph` zones will be created with the labels of value `test-storage-system-x3000`.
+* If the `spec.services.k8s_zone_prefix` has no value defined and the rack-id is `x3000`, then the `Kubernetes` zones will be created with the labels of value `x3000`.
+* If the `spec.services.ceph_zone_prefix` has no value defined and the rack-id is `x3000`, then the `Ceph` zones will be created with the labels of value `x3000`.
+   
+### 3. Setup/ configure Rack Resiliency
+
+The RR solution includes Ansible plays for use by the [Configuration Framework Service (CFS)](../../glossary.md#configuration-framework-service-cfs).
+These plays are deployed as part of the [`update_cfs_config` stage](../iuf/stages/update_cfs_config.md)
+using the [Install and Upgrade Framework (IUF)](../../glossary.md#install-and-upgrade-framework-iuf).
+It is initiated during [`management_nodes_rollout` stage](../iuf/stages/management_nodes_rollout.md) of IUF in order to enable/configure RR with placement discovery, placement validation, zoning (`kubernetes` zoning for master and worker nodes and Ceph zoning for Utility storage nodes) and applying `kyverno` policy automatically during boot time.
+
+If placement validation criteria is met as explained in the solution overview section "Validate distribution of management/ managed nodes", RR anisble plays will further configure RR zones (`kubernetes` and `Ceph`) and apply `kyverno` policy otherwise `ansible` plays will exit the RR configuration with respective placement validation failure message.

```

Comment:
```suggestion
If placement validation criteria is met as explained in the solution overview section "Validate distribution of management/ managed nodes", then RR Anisble plays will further configure RR zones (`kubernetes` and `Ceph`) and apply the `kyverno` policy; Otherwise, the Ansible plays will fail with a placement validation error message.

```